var/spack/repos/builtin/packages/py-radical-entk/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyRadicalEntk(PythonPackage):
+    """RADICAL Ensemble Toolkit is used for developing and executing
+    large-scale ensemble-based workflows."""
+
+    homepage = 'https://radical-cybertools.github.io'
+    git      = 'https://github.com/radical-cybertools/radical.entk.git'
+    pypi     = 'radical.entk/radical.entk-1.6.7.tar.gz'
+
+    maintainers = ['andre-merzky']
+
+    version('devel', branch='devel')
+    version('1.6.7', sha256='9384568279d29b9619a565c075f287a08bca8365e2af55e520af0c2f3595f8a2')
+
+    depends_on('py-radical-utils',       type=('build', 'run'))
+    depends_on('py-radical-saga',        type=('build', 'run'))
```

Comment:
I don't see saga listed as a dep

Reply:
right, this was set as an explicit dependency - will remove it (EnTK doesn't make explicit calls for SAGA)